hw7/Advanced_Task_2.ts

```diff
@@ -0,0 +1,28 @@
+// Advanced
+// Task 2
+// ```javascript
+//     const matrix = [
+//         [12, 98, 78, 65, 23],
+//         [54, 76, 98, 43, 65],
+//         [13, 324, 65, 312],
+//         [9092, 22, 45, 90000],
+//     ]
+// > Выведите в консоль среднее значение чисел в многомерном массиве.
+
+const matrix: number [] [] = [
+        [12, 98, 78, 65, 23],
+        [54, 76, 98, 43, 65],
+        [13, 324, 65, 312],
+        [9092, 22, 45, 90000],
+]
+
+// @ts-ignore
+let joinedMatrix: number [] = [].concat(matrix[0], matrix[1], matrix[2], matrix[3]) // как убрать ошибку?
```

Comment:
I suggest to substitude it with` let joinedMatrix: number [] = matrix[0].concat(matrix[1], matrix[2], matrix[3])`

Instead of default empty array you will join all arrays to the first one.

Reply:
Looks now, thank you :)